.github/workflows/build-and-deploy-prod.yml

```
@@ -0,0 +1,35 @@
+# This workflow will build a docker image, push it to ghcr.io, and deploy it to an Azure WebApp.
+name: Build and Deploy -- PROD
+
+on:
+  workflow_dispatch:
+  release:
+    types: [published]
+
+jobs:
+  determine-trigger:
+    name: Determine if this was triggered by a release or workflow_dispatch
+    runs-on: ubuntu-latest
+    outputs:
+      is-release: ${{ env.IS_RELEASE }}
+    steps:
+      - name: Check if this was triggered by a release
+        id: release
+        run: |
+          echo "IS_RELEASE"=${{ github.event_name == 'release' }} >> $GITHUB_ENV
+
+  build-and-deploy-prod:
+    uses: clearlydefined/operations/.github/workflows/app-build-and-deploy.yml@v1.0.0
+    needs: determine-trigger
```

Latest reusable workflow version that does the `determine-trigger` check in the shared workflow instead of the caller.
```suggestion
    uses: clearlydefined/operations/.github/workflows/app-build-and-deploy.yml@v1.1.0
```

With determine-trigger removed, the `needs` statement also has to be removed.